upf/upf.h

```diff
@@ -785,6 +792,9 @@ typedef struct
 
   f64 unix_time_start;
 
+  u8 imsi_len;
+  u8 imsi[8];
```

Comment:
additional fields from IMEI and MSISDN would be needed as well. Wouldn't it be simpler to put a reference to or a copy of the PFCP User Id IE here?

Reply:
Fixed, now it has PFCP User ID structure here